resources/js/app.js

```diff
@@ -6,6 +6,7 @@ require("jquery");
 require("./bootstrap");
 require("awesomplete/awesomplete");
 require("leaflet/dist/leaflet.js");
+const Chart = require("chart.js");
```

Comment:
Wird damit ChartJS auf jeder Seite geladen? Unnötig fast überall außer bei der Statistikseite..

Reply:
Kann man das überhaupt selektiv in einer großen app.js unterscheiden?

Reply:
Man könnte noch eine chart.js kompilieren lassen, die man dann nur selektiv einbindet.